src/components/KeyboardNavigation/__tests__/Provider.spec.js

```diff
@@ -0,0 +1,140 @@
+import React from "react";
+import { renderIntoDocument, cleanup, fireEvent } from "react-testing-library";
+
+import Provider from "../Provider";
+import { Consumer } from "../Context";
+
+const TesingConsumers = (
+  { children, index, datatestid } // eslint-disable-line
+) => (
+  <Consumer>
+    {({ focused }) => (
+      <div data-isfocues={focused === index} data-testid={datatestid}>
+        {children}
+      </div>
+    )}
+  </Consumer>
+);
+
+describe("KeyBoardProvider up", () => {
+  afterEach(cleanup);
+
+  it("Testing Keyboard accesability: Up outside range of number of Children ", () => {
```

Comment:
You can remove `Testing Keyboard accesability:` from all these tests we are already in the context of a test since this file is under `__tests__` & the file name includes `spec`